src/base/projects/Commit.svelte

```diff
@@ -26,6 +26,7 @@
     padding: 1rem;
     background: var(--color-foreground-background-subtle);
     border-radius: var(--border-radius);
+    margin-bottom: 1.5rem;
```

Comment:
Is this a visual change?

Reply:
Only on the patch page, on the commit page the spacing remains the same.

The `<Changeset>` component had the margin inside it originally, I extracted it out and adjusted the parent elements, so visually it looks the same. Now we have more control over the positioning over the component in other places in the code.

Commit view (no visual changes):
<img width="1728" alt="Screenshot 2022-08-30 at 14 09 32" src="https://user-images.githubusercontent.com/158411/187433170-bfea3fdc-dc4e-4b58-b19f-708397f4accc.png">

Adjusted the spacing in the patch view to look more natural (left is the proposed change):
<img width="1728" alt="Screenshot 2022-08-30 at 14 10 07" src="https://user-images.githubusercontent.com/158411/187433190-b0e50024-73ba-470b-8516-13d8ad9122c0.png">


The spacing difference was more obvious when switching the tabs:

https://user-images.githubusercontent.com/158411/187440138-d999d08c-d6fc-42fd-886c-b833e66b8e88.mov

vs now:

https://user-images.githubusercontent.com/158411/187440243-d7454b04-2243-4080-8966-e9ee56799468.mov